src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -587,6 +587,7 @@ public abstract class GenerateProtoTask extends DefaultTask {
   void compile() {
     Preconditions.checkState(state == State.FINALIZED, 'doneConfig() has not been called')
 
+    project.delete(outputBaseDir)
```

Comment:
Does it delete only own output dir? Or the root dir for all generateProto tasks?

Reply:
This is just its own individual root. ProtoPlugin appends `/${sourceSetOrVariantName}` to the value used here, so it is its own copy.